arrows/core/metadata_map_io_csv.cxx

```diff
@@ -222,7 +230,8 @@ bool
 metadata_map_io_csv
 ::check_configuration( VITAL_UNUSED vital::config_block_sptr config ) const
```

Comment:
If this is only going to be used for the export frequencies, could it be renamed to check_export_configuration or something similar?

Reply:
This function is virtual; it overrides `check_configuration()` in the base `algorithm` class  Therefore, it has to be named identically.